src/functionalTest/resources/scenarios/RIA-3309-notification-for-hearing-edit-case-listing-decision.json

```diff
@@ -0,0 +1,70 @@
+{
+  "description": "RIA-3309 Sends edit case listing notification to Home office API for state decision",
+  "disabled": false,
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "eventId": "editCaseListing",
+      "state": "decision",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "1111-2222-3333-4444",
+          "appealReferenceNumber": "DC/50001/2020",
+          "decisionHearingFeeOption": "decisionWithHearing",
+          "hearingDocuments": [],
+          "listCaseHearingCentre": "coventry",
+          "listCaseHearingLength": "60",
+          "listCaseHearingDate": "2018-12-31T12:34:56",
+          "listCaseRequirementsVulnerabilities": "something around Vulnerabilities",
+          "listCaseRequirementsMultimedia": "something around Multimedia",
+          "listCaseRequirementsSingleSexCourt": "something around SingleSexCourt",
+          "listCaseRequirementsInCameraCourt": "something around InCameraCourt",
+          "listCaseRequirementsOther": "something around Other",
+          "uploadAdditionalEvidenceActionAvailable": "Yes",
+          "ariaListingReference": "LP/12345/2019",
+          "witnessCount": "2",
+          "witnessDetails": [
+            {
+              "id": "1",
+              "value": {
+                "witnessName": "WitnessA"
+              }
+            },
+            {
+              "id": "2",
+              "value": {
+                "witnessName": "WitnessB"
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "homeOfficeReferenceNumber": "1111-2222-3333-4444",
+        "appealReferenceNumber": "DC/50001/2020",
+        "hearingDocuments": [],
+        "decisionHearingFeeOption": "decisionWithHearing",
+        "listCaseHearingCentre": "coventry",
+        "listCaseHearingLength": "60",
+        "listCaseHearingDate": "2018-12-31T12:34:56",
+        "listCaseRequirementsVulnerabilities": "something around Vulnerabilities",
+        "listCaseRequirementsMultimedia": "something around Multimedia",
+        "listCaseRequirementsSingleSexCourt": "something around SingleSexCourt",
+        "listCaseRequirementsInCameraCourt": "something around InCameraCourt",
+        "listCaseRequirementsOther": "something around Other",
+        "uploadAdditionalEvidenceActionAvailable": "Yes",
+        "ariaListingReference": "LP/12345/2019",
+        "homeOfficeHearingInstructStatus": "OK"
```

Comment:
I mentioned earlier about having a separare flag. do you think it is not needed? The same comment applies to all tests and handler

Reply:
No, I don't think there is a need for separate flag.

Reply:
But then there is no way of knowing if there were any edit on listing and what is the notification status for that isn't it? I think it is good to have a status flag. Also as I mentioned earlier if we have to display a message to TCW we need a flag to differentiate as well. Let me know if you still think it is not needed and we can discuss

Reply:
New status homeOfficeEditListingInstructStatus created